runtime/jcl/common/com_ibm_oti_vm_VM.c

```diff
@@ -46,7 +62,7 @@ Java_com_ibm_oti_vm_VM_getVMArgsImpl(JNIEnv *env, jobject recv)
 	/* Count only options that begin with "-" */
 
 	for (i = 0; i < originalSize; ++i) {
-		if ('-' == options[i].optionString[0]) {
+		if (('-' == options[i].optionString[0]) && (!getUserArgsOnly || IS_USER_ARG( j9vm_args, i ))) {
```

Comment:
Minor nitpick: `IS_USER_ARG( j9vm_args, i )` is how macros were used with the old coding guidelines, typically for new code we just do `IS_USER_ARG(j9vm_args, i)`